core/workflow-compiling-service/src/test/scala/edu/uci/ics/texera/service/resource/WorkflowCompilationResourceSpec.scala

```diff
@@ -203,9 +199,11 @@ class WorkflowCompilationResourceSpec extends AnyFlatSpec with BeforeAndAfterAll
     // verify the schema is correctly propagated for the final limit operator
     val compilationResult = assertSuccessfulCompilation(response)
     val finalLimitInputSchema =
-      compilationResult.operatorInputSchemas.get(limitOpDesc2.operatorIdentifier.id)
+      compilationResult.operatorOutputSchemas(filterOpDesc2.operatorIdentifier.id)
```

Comment:
is this test change necessary?

Reply:
Yes since we switched from input schemas to output schemas